src/env/builtins/lredir.c

```diff
@@ -543,6 +482,12 @@ int lredir_main(int argc, char **argv)
       return(0);
     }
 
+    /* check the MFS redirector supports this DOS */
+    if (!isInitialisedMFS()) {
+      printf("Unsupported DOS version\n");
```

Comment:
Since we still support the behind scenes initialisation of the redirector we don't rely upon emufs.sys being loaded. The only possible reason for isInitialisedMFS() to return false is a DOS variant not being supported by the redirector. I can add a log message to mfs.c, but would you prefer on init, on status query, or both?